development/full-kessel/.env

```
@@ -6,13 +6,13 @@ RBAC_IMAGE=quay.io/redhat-services-prod/hcc-accessmanagement-tenant/insights-rba
 
 ### SpiceDB schema
 # Default: download from stage rbac-config repo
-SCHEMA_ZED_URL=https://raw.githubusercontent.com/RedHatInsights/rbac-config/refs/heads/master/configs/stage/schemas/schema.zed
+SCHEMA_ZED_URL=https://raw.githubusercontent.com/project-kessel/rbac-config/refs/heads/master/configs/stage/schemas/schema.zed
 # Override with a local file path instead of downloading (leave empty to use URL)
 SCHEMA_ZED_FILE=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SCHEMA_ZED_FILE key should go before the SCHEMA_ZED_URL key

(UnorderedKey)